packages/fluxible-router/lib/handleHistory.js

```diff
@@ -251,7 +261,7 @@ function createComponent(Component, opts) {
                     }
                     break;
                 case TYPE_POPSTATE:
-                    if (options.enableScroll) {
+                    if (options.saveScrollInState && options.enableScroll) {
```

Comment:
We can skip `options.saveScrollInState` here and let it scroll to (0, 0) if enableScroll is true and saveScrollInState is false, to keep `saveScrollInState` option separate from `enableScroll` option.